src/core/theme/utils/typography.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  cssObjectsToCss,
+  FlattenSimpleInterpolation,
+} from '../../../utils/css/utils';
+import { typographyTokens, TypographyTokens } from '../typography';
+
+export type TypographyTokensAsCss = {
+  [key in keyof typeof typographyTokens]: string
+};
+
+type TypographyTokensAsCssProp = {
+  [key in keyof TypographyTokens]: FlattenSimpleInterpolation
+};
+export interface TypographyUtil extends TypographyTokensAsCssProp {}
+export class TypographyUtil {
+  static instance: any;
```

Comment:
No type for instance other than any?

Reply:
https://github.com/vrk-kpa/suomifi-ui-components/issues/193